### PR TITLE
Makes some plants a bit better

### PR DESCRIPTION
**Pull request by Lumipharon**
I have no clue what I'm doing so please bear with me.

Buffs various plants - most notably destroying angel and death berries.
Also adds a few basic chems to certain plants, to let botanists be ghetto chemists in some regards.

Some notable chems that this pr allows botany to produce includes mutagen (no more begging chem), meth and smoke.

Also adds me to admins.txt because that's what all the cool kids do.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
Some pretty controversial changes
:+1:

File: code/modules/hydroponics/grown/banana.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	icon_dead = "banana-dead"
 	genes = list(/datum/plant_gene/trait/slip)
 	mutatelist = list(/obj/item/seeds/banana/mime, /obj/item/seeds/banana/bluespace)
-	reagents_add = list("banana" = 0.1, "vitamin" = 0.04, "nutriment" = 0.02)
+	reagents_add = list("banana" = 0.1, "potassium" = 0.1, "vitamin" = 0.04, "nutriment" = 0.02)
```
Comment (Iamgoofball) [inline code/modules/hydroponics/grown/banana.dm]: who the fuck removed the potassium from bananas

**Comment by Iamgoofball** [comment]
:+1: it works

File: code/modules/hydroponics/grown/corn.dm [hunk 2]
```diff
@@ -13,7 +13,7 @@
 	icon_grow = "corn-grow" // Uses one growth icons set for all the subtypes
 	icon_dead = "corn-dead" // Same for the dead icon
 	mutatelist = list(/obj/item/seeds/corn/snapcorn)
-	reagents_add = list("cornoil" = 0.1, "vitamin" = 0.04, "nutriment" = 0.1)
+	reagents_add = list("cornoil" = 0.2, "vitamin" = 0.04, "nutriment" = 0.1)
```
Comment (newfren) [inline code/modules/hydroponics/grown/corn.dm]: :+1:

Comment (Lumipharon) [inline code/modules/hydroponics/grown/corn.dm]: Having to grow 100 corn at 100 potency just to make one shitty nade is pretty gay.

File: code/modules/hydroponics/grown/banana.dm (hunk 1 above)
Comment (Lumipharon) [inline code/modules/hydroponics/grown/banana.dm]: The mime, probably.

**Comment by Chocobro** [comment]
:+1:

File: code/modules/hydroponics/grown/corn.dm (hunk 2 above)
Comment (newfren) [inline code/modules/hydroponics/grown/corn.dm]: It's only like 20 corn but yeah this change is great.

**Comment by Lumipharon** [comment]
Merge or merge?

**Pull request merged**